docs/arch/adr-03.md

```diff
@@ -0,0 +1,86 @@
+>
+> Azure Blob Storage is useful for massive-scale, cloud-native applications that need to store unstructured data. To maximize performance and scale, Azure Blob Storage is a simpler storage abstraction than a true file system. You can access Azure Blob Storage only through REST-based client libraries (or directly through the REST-based protocol).
+>
+> Azure Files is specifically a file system. Azure Files has all the file abstracts that you know and love from years of working with on-premises operating systems. Like Azure Blob Storage, Azure Files offers a REST interface and REST-based client libraries. Unlike Azure Blob Storage, Azure Files offers SMB access to Azure file shares. By using SMB, you can mount an Azure file share directly on Windows, Linux, or macOS, either on-premises or in cloud VMs, without writing any code or attaching any special drivers to the file system. You also can cache Azure file shares on on-premises file servers by using Azure File Sync for quick access, close to where the data is used.
+
+### Option 1: Azure Blob Storage
+
+**Summary:** 
+Blob Storage is optimized for storing unstructured data: e.g. information that doesn't reside in a traditional row-column database.
+
+**Specs:**
+Target throughput for a single blob is up to 60 MiB per second
+
+Pros:
+- Blob Storage has been around for longer (appears to have shipped with the original launch of Azure Web Services in 2010), which means there will be more existing conversation around it (e.g. on stack overflow) and more tools/plugins for working with it
+- User reviews present Blob Storage as the go-to option, with File Services being an alternative that is employed for use cases that require specific additional functionality provided by File Services (e.g. mounting onto an existing file server, setting folder-specific permissions)
+- Offers a simpler, more basic solution
+- Blob Storage is much cheaper than file storage (approximately 1/5 of the cost per unit of data)
+- Greater maximum storage size than file storage (2PiB: 1 PiB = 2^50 bytes)
+
+Cons: 
+- Directory hierarchy system within blob storage is purely virtual - that is, a directory is merely an abstraction over the `/`-delimited names of the underlying container/blob hierarchy. In other words, a virtual directory is a prefix that you apply to a file (blob) name.
+- shared access signature permissions can only be granted in account level or container level. See [here](https://docs.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas) for details. This means that if we want to provide a link to the file, we will need to create a new blob container for the user so that the user isn't granted access to all transcription data. This may not turn out to be a real "downside" if it turns out there's no overhead involved in creating/deleting containers on demand.
```

Comment:
Looks like we can generate blob level SAS urls, not just container / account level permissions. https://docs.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas#specify-the-signed-resource-field
```
Specify b if the shared resource is a blob. 
This parameter grants access to the content and metadata of the blob.
```

Reply:
Ah, thanks for catching this! I must've misread something and come to the wrong conclusion. Ive updated the doc to account for this.